说文解字

xpc16 opened the pull request:
说文解字 (simple)

xpc16 committed:
说文解字

说文解字 (simple)

Berrysoft commented:
需要讨论几个问题：
* 为什么需要添加《说文》
* 能不能按照《汉语拼音方案》给拼音，而不是通过阿拉伯数字标注声调（我们支持UTF8）
* 部分拼音和反切不匹配
* 说文时代没有反切法，这里的反切的来源

xpc16 commented:
- 因为有建议添加说文，可能这能增加汉字知识
- 网上找的文库，没找到那种阿拉伯数字的
- 可能是古音的区别
- 可能来于《说文解字注》

Berrysoft commented:
最近有点忙，没空找，但是建议删除反切，修复拼音。我猜应该有一些库能够解决拼音的问题，有空的话也可以自己写一个。

Commit:
Curate Chinese fortunes and restore sign modes

xpc16 committed:
Curate Chinese fortunes and restore sign modes (#1)

Validated corpus cleanup, curated additions, generated indexes, and portable wrappers.

Commit:
Find fortune in standard games directories

xpc16 committed:
Find fortune in standard games directories (#2)

Use explicit /usr/games and /usr/local/games fallbacks after PATH lookup.

Commit:
Correct two Tang poem variants

xpc16 committed:
Correct two Tang poem variants (#3)

Use the verified full-text readings for future untruncated sign-in messages.
